scripts/build-src-containers.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+echo "INFO: Build sources containers"
+if [[ -z "${REPODIR}" ]] ; then
+  echo "ERROR: REPODIR Must be set for build src containers"
+  exit 1
+fi
+
+BUILDSH=${REPODIR}/contrail-container-builder/containers/build.sh
```

Comment:
let use uppercase for env variables that might set outside,
for local variables lowercase